rest/data/host.go

```diff
@@ -104,7 +103,12 @@ func GenerateHostProvisioningScript(ctx context.Context, env evergreen.Environme
 			Message:    errors.Wrap(err, "generating Jasper credentials").Error(),
 		}
 	}
-	script, err := h.GenerateUserDataProvisioningScript(ctx, env.Settings(), creds)
+	var githubAppToken string
+	var moduleTokens []string
+	if h.ProvisionOptions != nil && h.ProvisionOptions.TaskId != "" {
+		githubAppToken, moduleTokens = units.GetGithubTokensForTask(h.ProvisionOptions.TaskId, ctx)
+	}
```

Comment:
Yeah I agree that it's not ideal. But it does cause a package cycle and I don't think it's worth decoupling GenerateUserDataProvisioningScript from host struct for this, especially since we would also still need to call the function in fetchRemoteTaskData in units. 
But I'll do it if you feel strongly about it. Lmk.

Reply:
Eh, it seems fine as it is. It's not used that much anyways.